packages/lb-components/src/views/MainView/index.tsx

```diff
@@ -25,6 +25,7 @@ import PreviewResult from '@/components/predictTracking/previewResult';
 interface IProps {
   path: string;
   loading: boolean;
+  enableColorPicker?: boolean;
```

Comment:
You need to define the varible in  `AppProps`(`packages/lb-components/src/App.tsx`)